druid/README.md

```diff
@@ -0,0 +1,120 @@
+# Agent Check: druid
+
+## Overview
+
+The Datadog Agent collects many metrics from _Druid_ via DogStatsD. Those Druid metrics are related to queries, ingestion, and coordination:
+
+* Query Metrics
+* SQL Metrics
+* Coordination
+* General Health
+* Sys
+
+And many more. See details [here][1].
+
+I addition to metrics, the Datadog Agent also sends a service check for Druid health.
+
+## Setup
+
+### Prerequisite
+
+Druid 0.16 or above is required for this integration to work properly.
+
+### Installation
+
+The druid check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
```

Comment:
Not true, as per line 38

Reply:
agreed, this sentence should be removed and instructions starting at line 38 should be inlined here since they are for installation process.

Reply:
I updated the doc for `Installation`. Could you have a look again. Thx